fr-FR.lproj/Localizable.strings

```diff
@@ -0,0 +1,369 @@
+/* 
+  Localizable.strings
+  Undecimus
+
+  Created by Brieuc Caillot on 8/26/19.
+  Copyright © 2019 Pwn20wnd. All rights reserved.
+*/
+
+"Bundled Resources version missing." = "La version des ressources groupées est manquante."
+"Unsupported" = "N'est pas prise en charge"
+"Restore RootFS" = "Restaurer RootFS"
+"Ready to restore RootFS" = "Prêt à restaurer RootFS"
+"Re-Jailbreak" = "Re-Jailbreak"
+"Ready to re-jailbreak" = "Ready to re-jailbreak"
+"Jailbreak" = "Jailbreak"
+"Ready to jailbreak" = "Prêt à jailbreak"
+"Error" = "Erreur"
+"Bundled Resources version is missing. This build is invalid." = "La version des ressources groupées est manquante. Le build est invalide."
+"Days" = "Jours"
+"Invalid Entry" = "Entrée non valide"
+"The boot nonce entered could not be parsed" = "Le méthode de signature au démarrage entrée n'a pas pu être analysé"
```

Comment:
Le boot nonce n'a pas pu être analysé et validé (jpense que c'est mieux, OU "Le boot nonce n'a pas pu être validé".

Reply:
"Bundled Resources version is missing. This build is invalid." = "La version bundle ressources  est manquante. Le build est invalide."
"Days" = "Jours"
"Invalid Entry" = "Entrée non valide"
"The boot nonce entered could not be parsed" = "Le boot nonce entrée au démarrage, n'es pas analysé"